html_text/html_text.py

```diff
@@ -218,6 +218,12 @@ def extract_text(html,
     """
     if html is None:
         return ''
+    no_content_nodes = (
+        lxml.html.HtmlComment,
+        lxml.html.HtmlProcessingInstruction
```

Comment:
oh wow, didn't realize there is one more node type, thanks for handling this 👍

Reply:
There is one more (HtmlEntity). But I'm not sure how to get it in isolation; `fragment_fromstring("&nbsp;")` raises an exception.